docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -48,3 +48,11 @@ sent in parallel for each recovery. Defaults to `2`.
 +
 You can increase the value of this setting when the recovery of a single shard
 is not reaching the traffic limit set by `indices.recovery.max_bytes_per_sec`.
+
+`indices.recovery.max_concurrent_operation_chunks`::
+(<<cluster-update-settings,Dynamic>>, Expert) Number of operation chunk requests
+sent in parallel for each recovery. Defaults to `1`.
++
+Increasing the value of this setting with care as it can cause recoveries to consume
```

Comment:
Maybe reword this a little to something like (though maybe @DaveCTurner can use his native speaker powers here to help with wording :)):

Increase the value of this setting with care as higher values will cause increased CPU use, which might negatively impact indexing and search activities.

Reply:
> though maybe @DaveCTurner can use his native speaker powers here to help with wording

++

Reply:
Sure, how about this?

```
Concurrently replaying operations during recovery can be very
resource-intensive and may interfere with indexing, search, and other
activities in your cluster. Do not increase this setting without carefully
verifying that your cluster has the resources available to handle the extra
load that will result.
```

Reply:
Thanks David. Applied in https://github.com/elastic/elasticsearch/pull/58018/commits/e4f1fbfed406b007d3cc24b18781ce1b36607120.